packages/web-frontend/src/containers/Map/AreaTooltip.js

```diff
@@ -35,7 +35,8 @@ export class AreaTooltip extends Component {
   }
 
   render() {
-    const { permanent, onMouseOver, onMouseOut, text, sticky } = this.props;
+    const { permanent, onMouseOver, onMouseOut, texts, sticky } = this.props;
+    const textList = texts.map(text => <span>{text}</span>);
```

Comment:
Hmmm I think we may be missing an opporuntity here to bring the `AreaTooltip` component more inline with the `MeasurePopup` component? These two components behave in similar ways, and I think it would be quite beneficial to align some consistency between them 👍 

Measure Popup works by looping through each series (measureOption) and rendering the formatted display value for that series. I think we could do the same here, with perhaps the one proviso that if there's only a single series, we just display it's value and not title. If we then changed the display type of this overlay to 'fractionAndPercentage' then this would result in something like:
```
[ 'South Australia': 10/50 = 20%]
```
Now admittedly, that's not nearly as pretty as what you've got currently, with there being a nice field for 'Respondents' and 'Flu Vaccinated with Fever and Cough'. However, if we are willing to switch those overlays over to report-server, then we can fairly simply add the 'Respondents' and 'Flu Vaccinated with Fever and Cough' as fields in the report, right?